include/fmt/color.h

```diff
@@ -630,6 +638,91 @@ inline auto format_to(OutputIt out, const text_style& ts, const S& format_str,
                     fmt::make_format_args<buffer_context<char_t<S>>>(args...));
 }
 
+template <typename Arg, typename Char>
+struct formatter<detail::styled_arg<Arg>, Char> : formatter<Arg, Char> {
+  template <typename FormatContext>
+  auto format(detail::styled_arg<Arg> const& arg, FormatContext& ctx) const
```

Comment:
I'm not sure about `const` on the right side, @vitaut was not very happy about that, IIRC 😏

Reply:
Good catch!